plugin/trino-lance/pom.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add a sample config file to [conf/environment/multinode-all/](https://github.com/trinodb/trino/tree/master/testing/trino-product-tests-launcher/src/main/resources/docker/trino-product-tests/conf/environment/multinode-all) and update `EnvMultinodeAllConnectors`